src/main/java/org/homeschoolpebt/app/cli/TransmitterCommands.java

```diff
@@ -1,27 +1,124 @@
 package org.homeschoolpebt.app.cli;
 
+import formflow.library.data.Submission;
+import formflow.library.data.SubmissionRepository;
+import formflow.library.data.UserFile;
+import formflow.library.data.UserFileRepositoryService;
+import formflow.library.pdf.PdfService;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+import org.homeschoolpebt.app.data.Transmission;
 import org.homeschoolpebt.app.data.TransmissionRepository;
 import org.homeschoolpebt.app.data.TransmissionRepositoryService;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.homeschoolpebt.app.upload.CloudFile;
+import org.homeschoolpebt.app.upload.ReadOnlyCloudFileRepository;
 import org.springframework.data.domain.Sort;
 import org.springframework.shell.standard.ShellComponent;
 import org.springframework.shell.standard.ShellMethod;
 
 @ShellComponent
 public class TransmitterCommands {
 
-  @Autowired
-  TransmissionRepositoryService transmissionRepositoryService;
+  private final TransmissionRepositoryService transmissionRepositoryService;
+  private final TransmissionRepository transmissionRepository;
+  private final SubmissionRepository submissionRepository;
+  private final PdfService pdfService;
 
-  @Autowired
-  TransmissionRepository transmissionRepository;
+  private final UserFileRepositoryService uploadedFileRepositoryService;
+  private final ReadOnlyCloudFileRepository fileRepository;
+
+  public TransmitterCommands(TransmissionRepositoryService transmissionRepositoryService, TransmissionRepository transmissionRepository, SubmissionRepository submissionRepository,
+    PdfService pdfService, UserFileRepositoryService uploadedFileRepositoryService, ReadOnlyCloudFileRepository fileRepository) {
+    this.transmissionRepositoryService = transmissionRepositoryService;
+    this.transmissionRepository = transmissionRepository;
+    this.submissionRepository = submissionRepository;
+    this.pdfService = pdfService;
+    this.uploadedFileRepositoryService = uploadedFileRepositoryService;
+    this.fileRepository = fileRepository;
+  }
```

Comment:
Just curious - any reason to do it this way rather than autowiring? (It is the same, right?)